### PR TITLE
Add additional URL Linking and Content Cards examples in Swift

### PR DESCRIPTION
**Pull request by Bucimis**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/advanced_use_cases/linking.md [hunk 1]
```diff
@@ -169,13 +188,31 @@ As of Braze iOS SDK v2.21.0, the SDK percent-encodes links to create valid `NSUR
...
 }
 ```
 
+{% endtab %}
+{% tab swift %}
+
+```
+  func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
+    let urlString = url.absoluteString.removingPercentEncoding
+    // handle urlString
```
[the remark this answers is not in the document]
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/advanced_use_cases/linking.md]: fixed

File: _docs/_developer_guide/platform_integration_guides/ios/advanced_use_cases/linking.md [hunk 2]
```diff
@@ -87,9 +90,25 @@ continueUserActivity:(NSUserActivity *)userActivity
...
 }
 ```
 
+{% endtab %}
+{% tab swift %}
+
+```
+func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
+  if (userActivity.activityType == NSUserActivityTypeBrowsingWeb) {
+    let url = userActivity.webpageURL
```
[the remark this answers is not in the document]
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/advanced_use_cases/linking.md]: fixed

**Comment by EmilyNecciai** [comment]
@cla-bot check

**Comment by EmilyNecciai** [comment]
@cla-bot check

**Comment by EmilyNecciai** [comment]
@cla-bot check

**Comment by EmilyNecciai** [comment]
@cla-bot check

**Comment by EmilyNecciai** [comment]
@cla-bot check

**Pull request merged**